Batch/src/V1/resources/batch_service_rest_client_config.php

```diff
@@ -96,29 +96,6 @@
             ],
         ],
         'google.longrunning.Operations' => [
-            'CancelOperation' => [
-                'method' => 'post',
-                'uriTemplate' => '/v1/{name=projects/*/locations/*/operations/*}:cancel',
-                'body' => '*',
-                'placeholders' => [
-                    'name' => [
-                        'getters' => [
-                            'getName',
-                        ],
-                    ],
-                ],
-            ],
-            'DeleteOperation' => [
-                'method' => 'delete',
-                'uriTemplate' => '/v1/{name=projects/*/locations/*/operations/*}',
-                'placeholders' => [
-                    'name' => [
-                        'getters' => [
-                            'getName',
-                        ],
-                    ],
-                ],
-            ],
```

Comment:
This type of change is not flagged by our breaking change detector, yet would introduce breaking changes for our customers. We may need an extra check in order to ensure something like this can't happen!

Reply:
Confirmed this change is no longer in the PR, so it can be merged safely

Reply:
I added https://github.com/googleapis/google-cloud-php/issues/6219 to track ensuring that these types of breaking changes are caught in the future